config/services/_main.yaml

```diff
@@ -20,6 +20,11 @@ services:
         # fetching services directly from the container via $container->get() won't work
         public: false
 
+        bind:
+            $projectDir: '%kernel.project_dir%'
```

Comment:
@lchrusciel It's here just to save time, but services arguments should be manually configured on the future bundle.